service/controller/resource/tccpn/spec.go

```diff
@@ -0,0 +1,100 @@
+package tccpn
+
+import (
+	"github.com/aws/aws-sdk-go/service/ec2"
+	infrastructurev1alpha2 "github.com/giantswarm/apiextensions/pkg/apis/infrastructure/v1alpha2"
+)
+
+const (
+	// asgMaxBatchSizeRatio is the % of instances to be updated during a
+	// rolling update.
+	asgMaxBatchSizeRatio = 0.3
```

Comment:
I am not sure ALL OF THIS is really used. Can we check if we can drop a couple of things here?

Reply:
the batch ratio is not needed as we need to set the ASG update policy a little bit different than for workers

same with the pause time

Reply:
I would probabbly check for usages and try delete as much. 

a lot of things needs to be changed but that is out of scope for this ( like proper health checks to k8s-api-healthz, proper update strategy for ASG masters and so on) so we can clear const later

Reply:
Have removed asgratio and pause time.

Reply:
And what is with all that other crap? I don't think most of the code is really necessary. Just out of the blue I searched the project for `defaultEBSVolumeSize` and this is not even used anywhere. So we can not even drop it from here but also from where you copied it in the first place. This here needs cleanup. Let's sort this in separate PRs.

Reply:
:+1: